extension/include/boost/di/extension/providers/runtime_provider.hpp

```diff
@@ -58,11 +60,21 @@ class runtime_provider : public config {
   };
 
  public:
+  template <class T>
+  using scope_traits = di::extension::shared_config::scope_traits<T>;
```

Comment:
It is better but I still a bit concerned about not being able to configure `runtime_provider`. May be we can have `runitme_provider` as a template to configure scope traits and policies?

Reply:
Done :+1: